hydra-plutus/src/Hydra/Contract/Commit.hs

```diff
@@ -116,15 +114,7 @@ validator (_party, headScriptHash, commit) consumer ScriptContext{scriptContextT
                 -- is `Initial`
                 Just Initial{} ->
                   case consumer of
-                    ViaAbort ->
-                      case commit of
-                        Nothing -> True
-                        Just Commit{preSerializedOutput} ->
-                          -- There should be an output in the transaction corresponding to this preSerializedOutput
-                          traceIfFalse "cannot find committed output" $
-                            preSerializedOutput `elem` (Builtins.serialiseData . toBuiltinData <$> txInfoOutputs txInfo)
-                    -- NOTE: In the Collectcom case the inclusion of the committed output 'commit' is
-                    -- delegated to the 'CollectCom' script who has more information to do it.
```

Comment:
This NOTE was helpful. Similarly, a note for the abort case that the Head script will do the check that committed funds are reimbursed.